packages/inter-protocol/test/auction/test-auctionContract.js

```diff
@@ -102,6 +102,7 @@ const makeAuctionDriver = async (t, customTerms, params = defaultParams) => {
     params,
   );
   const { timerService } = terms;
+  /** @type {MapStore<Brand, { setPrice: (Ratio) => void }>} */
```

Comment:
this was copied from `makeManualPriceAuthority` but they both have a bug in that function signature require a param name before the type. without it, the interpretation is `(Ratio: any)`.
```suggestion
  /** @type {MapStore<Brand, { setPrice: (r: Ratio) => void }>} */
```